pandoras_flask/app.py

```diff
@@ -0,0 +1,14 @@
+import flask
+import metrics
+
+ping_app = flask.Flask(__name__)
+metrics.setup(ping_app)
+
+
+@ping_app.route('/ping', methods=['GET'])
+def ping():
+    return 'Pong', 200
+
+
+if __name__ == '__main__':  # pragma: no cover
+    ping_app.run(debug=True, host='0.0.0.0', port=8040)
```

Comment:
It would probably be best practise to omit the host here. We'll be proxying the request via nginx directly to the socket on the machine anyway, so it shouldn't need to bind externally.

Reply:
This is for if you execute `app` as a script directly, e.g. for testing. I'll add a comment to that effect.